rioxarray/rioxarray.py

```diff
@@ -316,7 +328,7 @@ def write_crs(self, input_crs=None, grid_mapping_name=None, inplace=False):
         data_obj.coords[grid_mapping_name] = xarray.Variable((), 0)
         grid_map_attrs = pyproj.CRS.from_user_input(data_obj.rio.crs).to_cf()
```

Comment:
This line still takes some time. I am thinking about adding some kind of a global option to disable.